src/DockTabs.tsx

```diff
@@ -328,6 +329,7 @@ export class DockTabs extends React.PureComponent<Props> {
             renderTabBar={this.renderTabBar}
             activeKey={activeId}
             onChange={this.onTabChange}
+            popupClassName={groupClassNames(group)}
```

Comment:
Are you sure `Tabs` component has `popupClassName` prop? I'v checked and I think it has not.

Reply:
yea.. seems like the projeect uses [rc-tabs: 11.16.1](https://github.com/ticlo/rc-dock/blob/master/yarn.lock#L5004), which does have this [popupClassName](https://github.com/react-component/tabs/releases?page=2#:~:text=Compare-,v11.16.0,-feat%3A%20support%20popupClassName)